iptables_exporter.go

```diff
@@ -87,7 +86,8 @@ func (c *collector) Collect(metricChan chan<- prometheus.Metric) {
 	tables, err := iptables.GetTables()
 	duration := time.Since(start)
 	if err == nil && len(tables) == 0 {
-		err = errors.New("no output from iptables-save; this is probably due to insufficient permissions")
+		log.Warn("No iptables rules found. Exiting.")
```

Comment:
I'd prefer it fails.. and we just not install this if there's no iptables rules. And I prefer we fix the parser to NOT emit duplicate metrics as we've seen it do.